.travis.yml

```
@@ -0,0 +1,41 @@
+dist: xenial
+
+language: node_js
+
+node_js:
+  - "8"
```

probably need to start updating this to 12...

```suggestion
  - "12"
```

---

rgr, updated